app/views/courses/TeacherCoursesView.coco.coffee

```diff
@@ -124,7 +124,12 @@ module.exports = class TeacherCoursesView extends RootView
     @$('#video-modal').modal('show')
     image_src = e.target.src.slice(e.target.src.search('/images'))
     video = (Object.values(@videoLevels || {}).find((l) => l.thumbnail_unlocked == image_src) || {})
-    @$('.video-player')[0].src = if me.showChinaVideo() then video.cn_url else video.url
+    video_url = if me.showChinaVideo() then video.cn_url else video.url
```

Comment:
oh let's make this condition more globally, since chinaVideo doesn't support multiple captions now. let's just use video.cn_url and skip all below logics for `showChinaVideo()`